### ISSUE TITLE
Cannot Debug Ruby Cucumber Feature File - Eclipse Hanging at 27%

### ISSUE THREAD

**Issue by abrarkhan1234** [Alterhu/CucumberPeople#21]
Hi, I am trying to setup debugging my ruby cucumber feature files but after setting up my debug configuration i try to debug the feature but eclipse just hangs at 27%. In the console log i get the following output:

[Oct/23/2015 Fri at 22:53:06 PM(INFO)]: Launch Cucumber Ruby project now.. 
[Oct/23/2015 Fri at 22:53:07 PM(INFO)]: Fast Debugger (ruby-debug-ide 0.5.0, ruby-debug-base19x 0.11.31) listens on 127.0.0.1:58262

I do not understand why it is just hanging, can you please advise.

**Comment by abrarkhan1234** [comment]
Hi can you please help with this

**Comment by Alterhu** [comment]
yes, I'm trying to find the problem and will let ur know it soon .thanks
 for your email.

On Wed, Nov 11, 2015 at 1:14 AM, Abrar <notifications@github.com> wrote:

> Hi can you please help with this
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Alterhu/CucumberPeople/issues/21#issuecomment-155710979>
> .
>



-- 
*Best Regards*
* Alter.hu*
